src/main/java/agh/ics/oop/GrassField.java

```diff
@@ -0,0 +1,98 @@
+package agh.ics.oop;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.Random;
+import java.lang.Math;
+
+public class GrassField extends AbstractWorldMap {
+
+    private final int fieldsOfGrass;
```

Comment:
zbędny atrybut, wyliczamy to w konstruktorze i zapominamy (potem mamy listę z rozmiarem)